multi_folder_download.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import os
+import subprocess
+import sys
+import time
+from datetime import datetime
+from distutils.util import strtobool
+
+import django
+from dateutil.relativedelta import relativedelta
+from dateutil.tz import tzutc
+from django.contrib.gis.geos import GEOSGeometry
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'geospaas_processing.settings')
+
+import geospaas_processing.downloaders as downloaders
+from geospaas.catalog.models import Dataset
+
+
+def main_function(ar):
+    if strtobool(ar.rel_time_flag):
+        designated_start_time = datetime.now().replace(tzinfo=tzutc())+ relativedelta(
+            hours=-int(ar.rel_time_lag))
+        designated_end_time = datetime.now().replace(tzinfo=tzutc())
+    else:
+        designated_start_time = datetime.strptime(ar.start_time, "%Y.%m.%d:%H").replace(
+            tzinfo=tzutc())
+        designated_end_time = datetime.strptime(ar.end_time, "%Y.%m.%d:%H").replace(tzinfo=tzutc())
+    downloaders.DownloadManager.MAX_DOWNLOADS = 500
```

Comment:
This is now an argument in the constructor of `DownloadManager`.

Reply:
Done in the next commit